hack/e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# This script assumes its running from a pod and does the following:
+# * Processes the template to setup a local cincy instance
+# * Prepares a graph using test data
+# * Ensures generated graph is valid
+
+# Debug information
+oc whoami
+oc project
+
+# Apply oc template
+oc process ../dist/openshift/cincinnati.yaml \
```

Comment:
We do not need `-f` for pointing to template file name?

Reply:
Fixed, thanks!